src/coreclr/nativeaot/Runtime/DebugHeader.cpp

```diff
@@ -270,7 +270,6 @@ extern "C" void PopulateDebugHeaders()
     DotNetRuntimeDebugHeader.GlobalEntries = &s_GlobalEntries;
 
     static_assert(MethodTable::Flags::EETypeKindMask         == 0x00030000, "The debugging data contract has a hard coded dependency on this value of MethodTable::Flags. If you change this value you must bump major_version_number.");
-    static_assert(MethodTable::Flags::RelatedTypeViaIATFlag  == 0x00040000, "The debugging data contract has a hard coded dependency on this value of MethodTable::Flags. If you change this value you must bump major_version_number.");
```

Comment:
@mikem8361 do you know if snapshot parser has an actual dependency on this bit, or was it just added because it looked relevant? Asking because we never actually emitted in NativeAOT and don't want to rev the debug header if I can help it.

Reply:
I found the answer myself in an old version. It's used in `Validate`. We need to bump the version because the bit could be reused.

There actually isn't much compat impact because this flag was never emitted (and neither was ClonedEEType that went away last week but there was no `static_assert` to warn me). Both cloned MethodTables and MethodTable indirection through IAT were never emitted on NativeAOT and the constanst are just leftovers from .NET Native.